### ISSUE TITLE
[TEST]wrong unique constraints and primary keys count

### ISSUE THREAD

**Issue by liemdt1811** [darold/ora2pg#367]
Hi Darold,
When I test the schema before copy data, the unique constraints and primary keys count are correct.
```
[TEST UNIQUE CONSTRAINTS COUNT]
ORACLEDB:USER_TBL:1
POSTGRES:user_tbl:1
```
But after I copy the data and test again, the unique constraints and primary keys count are all zero.
```
[TEST UNIQUE CONSTRAINTS COUNT]
ORACLEDB:USER_TBL:1
POSTGRES:user_tbl:0
```

In found the query:
```
SELECT count(*)
FROM pg_class
  JOIN pg_index ON (pg_index.indexrelid=pg_class.relfilenode)
  JOIN pg_indexes ON (pg_class.relname=pg_indexes.indexname)
WHERE pg_class.relkind = 'i'
  AND pg_index.indisprimary
  AND pg_indexes.tablename=?
```

Shouldn't it <code>pg_index.indexrelid=pg_class.**oid** </code>  ??

Regards,

**Comment by darold** [comment]
The query is the same in both case so I don't think it is responsible of the issue. I found a bug on primary key vs unique index during schema export. There is case where ora2pg creates a unwanted unique index where a primary key is already defined, so maybe in your case you have enabled DROP_INDEXES for data import and at recreate time the bug is not present. I will fix this issue this week end.

**Comment by darold** [comment]
Hi,

Sorry for the delay, I finally found time to fix this issue with commit f45b553. Part "[TEST UNIQUE CONSTRAINTS COUNT]" will now only count unique constraint excluding primary keys.

Regards,

**Comment by liemdt1811** [comment]
Hi darold,
   Thanks for the fixing.
Regards,
